saleor/product/migrations/0128_auto_20201006_1248.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1 on 2020-10-06 12:48
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
We should add data migration and assign this permission to all groups with `MANAGE_PRODUCTS`